### ISSUE TITLE
not use Hex View memory

### ISSUE THREAD

**Issue by dinhvan115** [L-Spiro/MhsX#1]
whoa. i can't see memory hex view on your source code. MHS 6.1 is full option ?

**Comment by L-Spiro** [comment]
This is a new MHS and is still in progress.
You can get the old source code fro links posted here: http://memoryhacking.com/forums/viewtopic.php?f=1&t=8385&p=100689&hilit=Source+Code&sid=b249a1f7a676a05ba02384ecd894337c#p100689

**Comment by dinhvan115** [comment]
yes thank so much, i having test MHSx Source code. i will rebuild MHS old source.

**Comment by dinhvan115** [comment]
Hi L.Spiro .
I changed tittle window and classname for your source code old version.
but it not use string my set. i had find encrypt string on your source code. ENC_DECL CONST WCHAR * g_pwcDbgIrb ENC_END( L"\x4C\x65\x68\x57\x6D\x4E\x7B\x76\x76\x67\x56\x6A\x75\x72\x75\x67\x46\x77\x6D\x64\x6C\x6B\x72" );
all most anti hacker use scan tittle and class name to find tool.
Can you share me tool to change encrypt string your source code v6.1.
your tittle use L.Spiro is block on all anti hacker.
thank you so much.
